src/nxdoc/nuxeo-server/data-store/indexing-and-query/configuring-the-elasticsearch-mapping.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Configuring the Search Mapping
-description: This documentation page talks about the many aspects you can tune for improving the search experience for your users when it comes to searching the document repository index using the nuxeo-search-client-opensearch1 package.
+description: This documentation page talks about the many aspects you can tune for improving the search experience for your users when it comes to searching the document repository index using the nuxeo-search-client-opensearch package.
```

Comment:
The suggestion came from the fact that `nuxeo-search-client-opensearch` is not a package.
```suggestion
description: This documentation page talks about the many aspects you can tune for improving the search experience for your users when it comes to searching the document repository index using one of Nuxeo Search Client OpenSearch package.
```